plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -67,7 +67,7 @@ func (a *alwaysPullImages) Admit(attributes admission.Attributes) (err error) {
 }
 
 // NewAlwaysPullImages creates a new always pull images admission control handler
-func NewAlwaysPullImages() admission.Interface {
+func NewAlwaysPullImages() *alwaysPullImages {
```

Comment:
done to conform to two interfaces?

Reply:
Yes, but will probably change that again because unexported return types seem to be weird to some https://github.com/kubernetes/kubernetes/pull/54485.